src/utilities/data-formatter.ts

```diff
@@ -148,27 +159,27 @@ class DataFormatter {
     this.addSpecial('unrecognized');
   }
 
-  static fromAllPrintable(maxThreshold, values = ['\n']) {
+  static fromAllPrintable(maxThreshold: number, values = ['\n']) {
     for (let i = 32; i <= 126; i++) {
       values.push(String.fromCharCode(i));
     }
     return new DataFormatter(values, maxThreshold);
   }
 
-  static fromAllPrintableInputOutput(maxThreshold, values = ['\n']) {
+  static fromAllPrintableInputOutput(maxThreshold: number, values = ['\n']) {
```

Comment:
`values` is missing a type

Reply:
TS infers it based on the default parameter value to be string[] which I think is the correct type.

Reply:
We're trying to conform and be as verbose as possible.  I think my preference would be to imply as little as possible if we're going to fully adopt typescript.

Reply:
Done for all methods/functions in the data-formatter file